### PR TITLE
Changes at week 0

### PR DESCRIPTION
**Pull request by jnallard**

### CODE REVIEW

File: src/main/java/frc/robot/commands/SimpleRollerControl.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import frc.robot.subsystems.Launcher;
+
+/**
+ * A class for controlling the rollers (intake, feeder, and launcher) in a simple way. Each respective subsystem can optionally be added via chaining:
+ * 
+ * <code>
+ * new SimpleRollerControl().intake(m_intake, 0.5).feeder(m_feeder, 0.3);
+ * </code
+ */
+public class SimpleRollerControl extends Command {
```
Comment (jnallard) [inline src/main/java/frc/robot/commands/SimpleRollerControl.java:19]: I added this class to make it easier to add these controls in RobotContainer. 

In the future, this class should hopefully not be needed after we have our more complex commands that will also control height and tilt automatically.

File: src/main/java/frc/robot/commands/SimpleControl.java [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+import frc.robot.subsystems.Lift;
+
+/**
+ * A class for controlling the mechanisms (intake, feeder, and flywheels) in a simple way. Each respective subsystem can optionally be added via chaining:
+ * 
+ * <code>
+ * new SimpleControl().intake(m_intake, 0.5).feeder(m_feeder, 0.3);
+ * </code
+ */
+public class SimpleControl extends Command {
```
Comment (jnallard) [inline src/main/java/frc/robot/commands/SimpleControl.java:24]: I added this class to make it easier to add these controls in RobotContainer.

In the future, this class should hopefully not be needed after we have our more complex commands that will also control height and tilt automatically.

Comment (jnallard) [reply src/main/java/frc/robot/commands/SimpleControl.java:24]: I then got carried away and added the ability to call this class easily from the auto script...

**Review by Anthony-Sinclair — APPROVED** [review]
Reviewed as a team, "Have you turned off and on again?" Kenny - 2/19/2024

**Pull request merged**